curriculum/js1/7.js

```diff
@@ -5,9 +5,13 @@
  * @returns {boolean}
  */
 
-const solution = (a) => {
-  return 0;
-};
+ const solution = (num, counter=2) => {
+   if (num <= 1) { return false }
+   if (num === 2) { return true }
```

Comment:
this case is unnecessary. If num is 2, then the next case should catch that and return true.